src/plugins/mission/mission_impl.cpp

```diff
@@ -5,6 +5,7 @@
 #include <fstream> // for `std::ifstream`
 #include <sstream> // for `std::stringstream`
 #include <cmath>
+#include <ctime>
```

Comment:
Is this required?

Reply:
@julianoes good catch, actually it's not - I added it when considering implementing the wait until hour/min/sec params but then decided to not implement that for now.

Reply:
Ok, fixed :)